src/responder/common/cache_req/plugins/cache_req_group_by_id.c

```diff
@@ -64,6 +64,7 @@ cache_req_group_by_id_lookup(TALLOC_CTX *mem_ctx,
                              struct sss_domain_info *domain,
                              struct ldb_result **_result)
 {
+    cache_req_idminmax_check(data, domain);
```

Comment:
My understanding is that you want to check the return of this function and then decide what to do based on that.